### ISSUE TITLE
Razer Basilisk V3 Pro Support Please!

### ISSUE THREAD

**Issue by GameboyLuke** [Tekk-Know/RazerBatteryTaskbar#19]
↑

**Comment by Tekk-Know** [comment]
This should be supported, have you tried the most recent release?

**Comment by GameboyLuke** [comment]
Installed the newest version, the taskbar icon on hover simply states "Device Disconnected"

**Comment by Tekk-Know** [comment]
@GameboyLuke the 1.0.7 version had a bug that might have impacted you, you might try 1.0.8.

**Comment by GameboyLuke** [comment]
unfortuntely i still have the same result

**Comment by GameboyLuke** [comment]
wait, I got it working! the only problem is that I have to close Razer Synapse. Which is an issue, is there any way to make it work so that Razer Synapse & the Battery Taskbar can run at the same time?

**Comment by Tekk-Know** [comment]
I've not specifically had the issue of them fighting, mine both work.

**Comment by GameboyLuke** [comment]
![Screenshot_1](https://github.com/Tekk-Know/RazerBatteryTaskbar/assets/48414726/8913c56f-6f6a-4c91-818a-63046d402cb1)
![Screenshot_2](https://github.com/Tekk-Know/RazerBatteryTaskbar/assets/48414726/77ef2355-b964-4a4b-a5f7-39684b770e65)

yeah, not sure what to tell you, here's what happened when i restart the battery taskbar after closing synapse

**Comment by GameboyLuke** [comment]
i also started the battery app first, confirmed it was working, launched syanpse and after a minute or two it tells me the device is disconnected

**Comment by Tekk-Know** [comment]
I re-wrote the app using the standard usb class, you might see if that changes anything for you.
https://github.com/Tekk-Know/RazerBatteryTaskbar/releases/download/debug/razerbatterytaskbar-1.0.9.Setup.exe

**Comment by GameboyLuke** [comment]
![image](https://github.com/Tekk-Know/RazerBatteryTaskbar/assets/48414726/894904cf-da86-471c-b207-723b5caf4aa4)
![image](https://github.com/Tekk-Know/RazerBatteryTaskbar/assets/48414726/47ff2eab-2486-44cc-bb67-8af647cfc450)
![image](https://github.com/Tekk-Know/RazerBatteryTaskbar/assets/48414726/f486f58b-469c-44cf-a519-2e5afb538b09)

getting this error now, even with synapse closed

**Comment by kbpersonal** [comment]
For me, using 1.0.9, it still seems like unless all Razer apps are fully exited, the icon says 'Device disconnected' but once Razer isn't communicating with the device anymore it comes back to show the battery level automatically. This happens even when the taskbar shows the battery and then I launch Synapse.

**Comment by MozgC** [comment]
YEs, same with 1.0.9, if Synapse is running, this app says Device disconnected.
